README.md

```diff
@@ -419,11 +419,11 @@ Here are some companies/projects (alphabetical order) using OpenAPI Generator in
 - [Bithost GmbH](https://www.bithost.ch)
 - [GMO Pepabo](https://pepabo.com/en/)
 - [Raiffeisen Schweiz Genossenschaft](https://www.raiffeisen.ch)
+- [RepreZen API Studio](https://www.reprezen.com/swagger-openapi-code-generation-api-first-microservices-enterprise-development)
```

Comment:
@tedepstein one more suggestion. What about linking to https://www.reprezen.com instead as the [page](https://www.reprezen.com/swagger-openapi-code-generation-api-first-microservices-enterprise-development) doesn't mention OpenAPI Generator at all?